src/app.ts

```diff
@@ -235,14 +236,27 @@ export class OracleApplication {
           kit.addAccount(privateKey)
           this.config.address = privateKeyToAddress(privateKey)
         } else {
-          this.logger.info(`DEVMODE enabled, used mock address ${this.config.mockAccount}`)
           this.config.address = this.config.mockAccount
+          this.logger.info(`DEVMODE enabled, used mock address ${this.config.mockAccount}`)
         }
         break
+      case WalletType.NODE_ACCOUNT:
+          kit = newKit(httpRpcProviderUrl)
+          if (this.config.address){
+            kit.defaultAccount = this.config.address
+          } else {
+            // If not default address, use the first one of the account
+            const account = (await kit.web3.eth.getAccounts())[0]
+            kit.defaultAccount = account
+            this.config.address = account
+          }
```

Comment:
I think there's some extra indenting here -- running `yarn prettify` should fix both issues.